### DISCUSSION TITLE
Translate help

### DISCUSSION THREAD

**Discussion by loulou64490** [Jeffser/Alpaca#94] [Q&A]
Hello, I will use this tread to ask help about translation

**Comment by loulou64490** [comment]
First question, what's "override" ????

**Reply by Jeffser** [reply]
Oh sorry, I didn't see the notification, I'm not completely sure what "override" means (English is my second language too) I interpret it as overwriting a variable, you could just leave the word override as it is, that's what I did with spanish

**Reply by loulou64490** [reply]
Okay, that's what I had understand. But in french the word override is a false friend for the context, so I translated it parameters
